com.unity.render-pipelines.high-definition/Runtime/PostProcessing/PostProcessSystem.cs

```diff
@@ -3348,12 +3357,14 @@ FinalPassParameters PrepareFinalPass(HDCamera hdCamera, BlueNoise blueNoise, boo
         static void DoFinalPass(in FinalPassParameters  parameters,
             RTHandle                source,
             RTHandle                afterPostProcessTexture,
-            RenderTargetIdentifier  destination,
+            RTHandle                destinationHandle,
             RTHandle                alphaTexture,
             CommandBuffer           cmd)
         {
+
             // Final pass has to be done in a pixel shader as it will be the one writing straight
             // to the backbuffer eventually
+            RenderTargetIdentifier destination = destinationHandle;
```

Comment:
No need to do this intermediate step, you can get the rt directly from RTHandle

Reply:
Removed